lib/miq_helper.rb

```diff
@@ -0,0 +1,26 @@
+require 'pathname'
+require 'active_support/core_ext/object/blank'
+require 'active_support/string_inquirer'
+
+module Miq
```

Comment:
lib/manageiq specifically...so I'm expecting this to be in lib/manageiq.rb  (File name should match the class path)